Analysis/Physics/Chic2invi/Chic02ee/Chic02ee-00-00-01/src/Chic02ee.cc

```diff
@@ -306,7 +308,7 @@ CLHEP::Hep3Vector xorigin = getOrigin();
     // Good Kalman Track 
     if(!(*itTrk)->isMdcKalTrackValid()) continue;
 	cout<<"isMdcTrackValid"<<endl;
```

Comment:
after fixed the bug, u could remove or comment out the cout lines, otherwise the job will generate very large log file